content/tutorials/getting-started/index.md

```diff
@@ -149,12 +149,12 @@ Running a test locally is also a great way to do debugging before sending it to
 
 ## Step 3 - Generate a package {#generate-package}
 
-Gatling Enterprise requires a package built from a Gatling project in order to run a test. This package is called an `artifact`.
+Gatling Enterprise requires a package built from a Gatling project in order to run a test.
 
-To create an artifact, run the `artifact.bat` (if Windows) or `artifact.sh` (if Linux/MacOS) file, Gatling will start creating your first artifact:
+To create a package, run the `enterprisePackage.bat` (if Windows) or `enterprisePackage.sh` (if Linux/MacOS) file, Gatling will start creating your first package:
```

Comment:
Please revert all command changes. Those will only be release in Gatling 3.7, which should probably be out only end of September at best.